tests/copulas/multivariate/test_center_tree.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from unittest import TestCase
+
+import numpy as np
+import pandas as pd
+
+from copulas.multivariate.tree import CenterTree
+from copulas.univariate.kde import KDEUnivariate
+
+LOGGER = logging.getLogger(__name__)
+
+
+class TestCenterTree(TestCase):
+    def setUp(self):
+        self.data = pd.read_csv('data/iris.data.csv')
+        self.tau_mat = self.data.corr(method='kendall').values
+        self.u_matrix = np.empty([self.data.shape[0], self.data.shape[1]])
+        count = 0
+        for col in self.data:
+            uni = KDEUnivariate()
+            uni.fit(self.data[col])
+            self.u_matrix[:, count] = [uni.get_cdf(x) for x in self.data[col]]
+            count += 1
+        self.trees = []
+        self.trees.append(CenterTree(0, 4, self.tau_mat, self.u_matrix))
+
+    def test_first_tree(self):
+        self.assertEquals(self.trees[0].edges[0].L, 0)
+
+    def test_first_tree_likelihood(self):
+        print(self.trees[0])
```

Comment:
I ran it with pytest -s to get the stdout. What is the correct way if I want to debug when running using pytest (sth like print statement or debugger in a regular program)?

Reply:
@aliciasun The problem here is not about the debugging itself which you can do in any way you like (I personally put `ipdb.set_trace()` statements to go into interactive debuggin, but using print statemtents can also work).
The problem is, instead, about the fact that you left there the things that you are using for debugging (i.e. the print statement).

Think about it this way: Unit tests are somehting automated that needs to be run and validated by a machine (in this case Travis), so you can rely on looking at an output with your eyes to tell whether the test is good or not.

In other words: If this print has been put there to validate something by looking at the outputs, it should  be converted to the corresponding assertions for travis to be able to detect the errors.
Otherwise, if the print statement has been put there because you used it while debugging a strange behaviour that you could not understand otherwise, it should be removed.